### PR TITLE
Scenario cards + Website indentation fix

### LINKED ISSUE #6496: Scenario Cards for 1.5 features

### LINKED ISSUE #6510: [Website] Schema Explorer left nav is too indented
Compare against adaptivecards.io and see the difference.

image

---
* URL: http://adaptivecards-ci.azurewebsites.net/explorer/Action.OpenUrl.html

### PR DESCRIPTION
**Pull request by JeanRoca**
In this PR I added all scenario cards for 1.5 features. These include:
- An application login card to demonstrate text masking
- An input form with show card for both english and arabic to demonstrate RTL support
- An order confirmation page to demonstrate actionset overflow
- An order delivery page that demonstrates both tooltip and isEnabled
- Re-created the FlightUpdate card with the new Table element instead of column sets
- A restaurant order form with filtered choicesets.

This closes #6496 
I also fixed #6510 by reducing the padding in the CSS.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/6733)

### CODE REVIEW

**Review by licanhua — APPROVED** [review]

File: source/nodejs/adaptivecards-site/themes/adaptivecards/source/css/style.css [hunk 1]
```diff
@@ -320,7 +320,7 @@ ul.action-list {
 
 .toc-todo li,
 .toc-todo a {
-  padding-left: 30px;
+  padding-left: 15px;
```
Comment (almedina-ms) [inline source/nodejs/adaptivecards-site/themes/adaptivecards/source/css/style.css:323]: why is this change required?

**Review by almedina-ms — DISMISSED** [review]
:clock1:

File: source/nodejs/adaptivecards-site/themes/adaptivecards/source/css/style.css (hunk 1 above)
Comment (JeanRoca) [reply source/nodejs/adaptivecards-site/themes/adaptivecards/source/css/style.css:323]: This is how the website currently looks:
image

Comment (JeanRoca) [reply source/nodejs/adaptivecards-site/themes/adaptivecards/source/css/style.css:323]: This is how the change in the PR makes it look:
image

Issue #6510 states the current website is over indented. I reduced this indentation.

**Review by paulcam206 — APPROVED** [review]
